manifests/plugin/python/module.pp

```diff
@@ -0,0 +1,34 @@
+# Single module definition
+define collectd::plugin::python::module (
+  $config,
+  $script_source = undef,
+  $modulepath    = $collectd::plugin::python::modulepath,
+  $module        = $title,
+  $ensure        = 'present',
+  $order         = '50',
+  $python_conf   = "${collectd::params::plugin_conf_dir}/11-python-config.conf",
```

Comment:
I am not sure it's useful to have that parameter, why would we want to have different config files ? (and would it even be handled with the header and footer?)